.rubocop.yml

```diff
@@ -12,6 +12,14 @@ AllCops:
     # Apparently needed for Travis CI since it bundles into this directory
     - 'vendor/bundle/**/*'
 
+Rails:
```

Comment:
Since this is an over configuration setting rather than a cop setting, I moved it to the top.